html-syntax/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>HTML Syntax</title>
+  </head>
+  <body>
+    <img src="https://archives.bulbagarden.net/media/upload/thumb/7/7e/006Charizard.png/250px-006Charizard.png">
```

Comment:
just make sure to add the proper `alt` text!

Reply:
![image](https://user-images.githubusercontent.com/96906988/158252536-3bb62294-b3e9-42b3-a60d-f282013ac933.png)